disnake/ui/item.py

```diff
@@ -81,7 +81,7 @@ class Item(WrappedComponent, Generic[V_co]):
     The current UI items supported are:
 
     - :class:`disnake.ui.Button`
-    - :class:`disnake.ui.Select`
+    - subtypes of :class:`disnake.ui.BaseSelect` (:class:`~disnake.ui.Select`)
```

Comment:
this should display `ui.Select` *at minimum* and perhaps the full name.

Reply:
https://github.com/DisnakeDev/disnake/pull/800/commits/a6694525818becff893c2fb0909435f6f10fe863